README.md

```diff
@@ -1 +1,66 @@
-# microverse-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> This is a portfolio website set up for and by web developers.
+
+
+## Built With
+
+- HTML
+- CSS
+
+<!-- ## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com) -->
+
+
+<!-- ## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.** -->
+
+
+<!-- To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment -->
+
```

Comment:
- [ ] Kindly remove all commented code in your codebase to make it clean and maintainable.